meta-lmp-bsp/recipes-kernel/linux-firmware/linux-firmware_%.bbappend

```diff
@@ -6,6 +6,9 @@
 # sdiouart8987_combo_v0.bin is required for WiFi/BT support but this firmware
```

Comment:
There is a typo in the commit log.

s/neede/needed

Reply:
Fixed!  Thanks.